MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 include requirements.txt
 include LICENSE
 include NOTICES
+recursive-include mmf/configs/ *.yaml
```

Comment:
Can you also `recursive-include mmf/projects/ *.yaml`?

Reply:
right,  fixed.